shell/platform/android/platform_view_android_jni.cc

```diff
@@ -444,6 +446,52 @@ static void RegisterTexture(JNIEnv* env,
   );
 }
 
+static void RegisterShareTexture(JNIEnv* env,
+                                 jobject jcaller,
+                                 jlong shell_holder,
+                                 jlong texture_id,
+                                 jlong share_texture_id) {
+  ANDROID_SHELL_HOLDER->GetPlatformView()->RegisterExternalShareTexture(
+      static_cast<int64_t>(texture_id),       //
+      static_cast<int64_t>(share_texture_id)  //
+  );
+}
+
+static jobject GetShareContext(JNIEnv* env,
```

Comment:
Should this method have a try block to catch c++ exceptions and translate them to java exceptions?  There might be helpers for that.  I forget if we allow exceptions in our c++ code.

Reply:
it seems that exceptions was disabled. 
We got a compile error when we add a try block:
error: cannot use 'try' with exceptions disabled